dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProjectServiceImpl.java

```diff
@@ -97,8 +97,7 @@
                 .build();
 
         if (projectMapper.insert(project) > 0) {
-            Project insertedProject = projectMapper.queryByName(name);
-            result.put(Constants.DATA_LIST, insertedProject);
+            result.put(Constants.DATA_LIST, project);
             putMsg(result, Status.SUCCESS);
```

Comment:
hi, creat project success also need the project?

Reply:
https://github.com/apache/incubator-dolphinscheduler/issues/2619